src/backend/aspen/app/app.py

```diff
@@ -151,14 +151,14 @@ def setup_userinfo(user_id):
         sentry_sdk.capture_message(
             f"Requested auth0_user_id {user_id} not found in usergroup query."
```

Comment:
i think it would be nice to  also log their login email? That way we can more easily tell if we need to add a new user to our DB  or if we're getting unexpected login attempts (without involving detective @shanaxel42 lol)

Reply:
if we have that information in the request